gtsfm/view_graph_estimator/reprojection_error_view_graph_estimator.py

```diff
@@ -0,0 +1,328 @@
+"""
+In the spirit of "Growing Consensus" (Son16cvpr)
+https://www.cv-foundation.org/openaccess/content_cvpr_2016/papers/Son_Solving_Small-Piece_Jigsaw_CVPR_2016_paper.pdf
```

Comment:
Does this module operate on _N_ views as in the paper, or will it only work for 3-views? If the latter, maybe this should be omitted until later on.